internal/BUILD.bazel

```diff
@@ -14,6 +14,9 @@
 
 # gazelle:exclude worker_protocol.proto
 
+# BEGIN-DEV-ONLY
```

Comment:
We don't want any content from this file, right? How about we just generate an empty marker file in the build?